package.json

```diff
@@ -31,7 +31,7 @@
     "async": "^2.0.0",
     "loader-utils": "^1.0.0",
     "lodash": "^4.0.0",
-    "source-map": "^0.7.0",
+    "source-map": "^0.5.6",
```

Comment:
Maybe `^0.6.0` is ok here, according to the `CHANGELOG` support for `node < v8.0.0` was drop in `v0.7.0` 

https://github.com/mozilla/source-map/blob/master/CHANGELOG.md

> Breaking change: Drop support for Node < 8. If you want to support older versions of node, please use **v0.6** or below.

But the `CHANGELOG`  sadly misses notes about the `0.6.0` release 😞, so maybe `0.5.6` is 'safer'

Reply:
I figured since the project was previously on `^0.5.6`, it was safer to just revert this version bump. But I would be happy to upgrade `^0.6.0` instead.

Reply:
Yep, let's better leave it :)